packages/volto-iframe-block/src/components/View.jsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
 import cx from 'classnames';
-import { isValidUrl } from './schema';
+import React from 'react';
+
+import { isValidUrl } from '../helpers/isValidUrl';
```

Comment:
Use an absolute import so that it still works if this component is shadowed.

```suggestion
import { isValidUrl } from '@kitconcept/volto-iframe-block/helpers/isValidUrl';
```